tests/test_writer.py

```diff
@@ -1,50 +1,56 @@
-from pytest import fixture
+from pytest import yield_fixture
 
 from pip.index import FormatControl
-from piptools.utils import comment
+from piptools.utils import comment, UNSAFE_PACKAGES
 from piptools.writer import OutputWriter
 
 
-@fixture
+class Writer(object):
```

Comment:
I don't understand this refactoring of correct code.

For what it's worth, pytest now doesn't discriminate between a yield and an ordinary fixture. @fixture is only needed.